Tesla.css

```diff
@@ -0,0 +1,214 @@
+* {
+    margin: 0;
+    padding: 0;
+  }
+  
+  html,
+  body {
+    height: 100%;
+    width: 100%;
+  }
+  .body {
+    font-family: Montserrat;
+    font-weight: 600;
+    font-style: normal;
+    margin: 0;
+}
+.header {
+    position: relative;
+    font-family: Montserrat;
+    font-weight: 600;
+    font-style: normal;
+}
+.logo {
+    width: 166px;
+    height: 51px;
+    margin-left: 42.72px;
+    margin-top: 24px;
+    margin-right: 258.14px;
+}
+.link {
+    text-decoration: none;
+    color: #000000;
+}
+.nav__link {
+    font-size: 15px;
+    margin-top: 40px;
+    width: 62px;
+    height: 19px;
+    font-size: 15px;
+}
+.nav__link-s {
+    color: #E51937;
+    margin-right: 63.14px ;
+    margin-bottom: 5px;
+}
+.nav__link-x {
+    margin-right: 24.86px;
+}
+.nav__link-3 {
+    margin-right: 44px;
+}
+.nav__link-road {
+    margin-right: 46px;
+}
+.nav__link-energy {
+    margin-right: 221.14px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)